package/zshelf/package

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(zshelf)
+pkgdesc="Z-Library browser and downloader"
+url=https://github.com/khanhas/zshelf
+pkgver=0.1.0
+section=utils
+timestamp=2021-01-30T15:48Z
+maintainer="khanhas <xuankhanh963@gmail.com>"
+license=GPL-3.0
+depends=(node)
+
+source=(
+    https://github.com/khanhas/zshelf/archive/v0.1.0.zip
+    zshelf.draft
+    zshelf.png
+)
+sha256sums=(
+    95252da324dbaec419f277532afe29c75cbac901a7a5da828ea6abb620687611
+    SKIP
+    SKIP
+)
+
+image=qt:v1.1
+build() {
+    qmake zshelf.pro
+    make
+
+    # Get needed build packages
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update -y
+    apt-get install -y \
+        --no-install-recommends \
+        -o Dpkg::Options::="--force-confdef" \
+        nodejs npm
+
+    cd backend
+    npm install
+}
+
+package() {
+    install -d "$pkgdir"/opt/share/zshelf
+
+    cp -r "$srcdir"/backend "$pkgdir"/opt/share/zshelf/backend
+    cp "$srcdir"/config.json "$pkgdir"/opt/share/zshelf
+    install -D -m 755 -t "$pkgdir"/opt/share/zshelf "$srcdir"/zshelf
+
+    ln -s "$pkgdir"/opt/share/zshelf/zshelf "$pkgdir"/opt/bin
+
+    install -D -m 644 -t "$pkgdir"/opt/etc/draft "$srcdir"/zshelf.draft
+    install -D -m 644 -t "$pkgdir"/opt/etc/draft/icons "$srcdir"/zshelf.png
+}
+
+configure() {
+    echo ""
+    echo "To download books, you have to configure domain and cookie first."
+    echo "Please follow instructions on:"
+    echo "https://github.com/khanhas/zshelf#important-configure-domain-and-cookie"
+    echo "Config file locates at:" "$pkgdir"/opt/share/zshelf/config.json
```

Comment:
The `"$pkgdir"` also seems not necessary here as this code will run on the device and the `$pkgdir` doesn't exist there.

Reply:
I can do this 👍